src/rpc/calls/protocolVersion.ts

```diff
@@ -0,0 +1,45 @@
+import { RPCError, RPCRequest, RPCResponse } from '../../types/types'
+import { callStarknet } from '../../utils/callHelper'
+
+export async function protocolVersionHandler(
+  request: RPCRequest,
+): Promise<RPCResponse | RPCError> {
+  if (request.params.length != 0) {
+    return {
+      jsonrpc: request.jsonrpc,
+      id: request.id,
+      error: {
+        code: -32602,
+        message: 'Invalid argument, Parameter field should be empty.',
+      },
+    }
+  }
+
+  const response: RPCResponse | string = await callStarknet('testnet', {
+    jsonrpc: request.jsonrpc,
+    method: 'starknet_specVersion',
+    params: [],
+    id: request.id,
+  })
+
+  if (
+    typeof response === 'string' ||
+    response === null ||
+    typeof response === 'undefined'
+  ) {
+    return {
+      jsonrpc: request.jsonrpc,
+      id: request.id,
+      error: {
+        code: -32602,
+        message: response,
```

Comment:
Starknet response is the string of starknet version with decimal points. Like "0.5.1", however ethereum rpc returns hex decimal.

So we need to return hex value. Like following algorithm
Each dot represents two numbers so
XX.YY.ZZ => XXYYZZ and then this value has to be converted into hex format then return.

For example 
"0.5.1" => 051 = 51 => 0x33

"0.6.22" => 622 = 0x62E

Conflict can happen in cases like 0.5.22 and 0.52.2 however it is not really big deal at the moment. We will find proper way future, this call is not that necessary.